core/internal/cltest/util.go

```diff
@@ -11,3 +14,24 @@ func MustHexDecodeString(s string) []byte {
 	}
 	return a
 }
+
+// CommitLoop creates a goroutine that repeatedly commits and returns a
+// cancellation function to avoid leaking goroutines
+func CommitLoop(b *backends.SimulatedBackend) func() {
```

Comment:
this already exists in `core/internal/cltest/simulated_backend.go`, the `Mine()` function